src/content/news/2025-10-04-maplibre-newsletter-september-2025/index.mdx

```diff
@@ -0,0 +1,172 @@
+---
+title: "MapLibre Newsletter September 2025"
+date: "2025-10-04"
+categories: ["newsletter"]
+authors:
+  [
+    harel-mazor,
+    bart-louwers,
+    birk-skyum,
+    joscha-eckert,
+    frank-elsinga,
+    yuri-astrakhan,
+    ramya-ragupathy,
+  ]
+draft: false
+---
+
+import { Image } from "astro:assets";
+
+Welcome to the MapLibre Newsletter, September 2025 edition!
+
+This edition features advances in MapLibre Native, GL JS, Flutter, Martin, and more. Plus, we’ve given [our roadmap section](https://maplibre.org/roadmap/) a fresh redesign. Go check it out to see what’s in progress, what’s under consideration, and what’s been released.
+
+A big thanks to [Birk Skyum](https://maplibre.org/about/birk-skyum/), [Frank Elsinga](https://maplibre.org/about/frank-elsinga/) and [Yuri Astrakhan](https://maplibre.org/about/yuri-astrakhan/) for their continued leadership, contributions, and support in driving these initiatives forward. 🙌
```

Comment:
Lets not do that, unless @birkskyum wants to be mentioned ^^

```suggestion
```

If at all, this is birks' work

```suggestion
A big thanks to [Birk](https://maplibre.org/about/birk-skyum/) for driving this initiative forward. 🙌
```